scripts/ei/deploy.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+# -------------------------------------------------------------------------------------
+# Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# --------------------------------------------------------------------------------------
+
+deploymentName=$1; shift
+cloudformationFileLocations=("$@")
+
+deploymentDirectory="${WORKSPACE}/deployment/${deploymentName}"
+parameterFilePath="${deploymentDirectory}/parameters.json"
+outputFile="${deploymentDirectory}/deployment.properties"
+currentScript=$(dirname $(realpath "$0"))
+
+source ${currentScript}/../common-functions.sh
+
+counter=0
+stackName=$(extractParameters "StackName" ${parameterFilePath})
+tempStackName=${stackName}
+for cloudformationFileLocation in ${cloudformationFileLocations[@]}
+do
+    parameterFilePath="${deploymentDirectory}/parameters.json"
+    # When one deployment depends on several CFNs then the stackname will be appened
+    # with an itteration number
+    if [[ ${#cloudformationFileLocations[@]} -gt 1 ]];
+    then
+        stackName="${tempStackName}-${counter}"
+    fi
+
+    if [[ ${cloudformationFileLocation} == *"sample"* ]];
```

Comment:
I just had a look at the CFN files. I could find that there are sample backend files.  But these are named as `activemq-be.yaml` and `vfs-be.yaml`. It does not have the name `sample` in it. So if we are using both these CFN files, shall we introduce a new logic to get these deployed?

Reply:
We no longer use these backends. Therefore we do not need these backend files.